models/Recording.js

```diff
@@ -193,10 +193,10 @@ module.exports = function(sequelize, DataTypes) {
 
 
   // local
-  var handleTagMode = (tagMode, tagWhats) => {
+  var handleTagMode = (tagMode, tagWhatsIn) => {
+    const tagWhats = (tagWhatsIn && (tagWhatsIn.length > 0)) ? tagWhatsIn : null;
     if (!tagMode) {
-      const hasWhatTags = (tagWhats && (tagWhats.length > 0));
-      tagMode = (hasWhatTags) ? 'tagged' : 'any';
+      tagMode = (tagWhats) ? 'tagged' : 'any';
     }
 
     const humanSQL = 'NOT "Tags".automatic';
```

Comment:
Could remove the extra brackets in lines 204 - 226
 ```return '(' + tagOfType(tagWhats, humanSQL) + ') AND (' + tagOfType(tagWhats, AISQL) + ')';```
could be
```return  tagOfType(tagWhats, humanSQL) + ' AND ' + tagOfType(tagWhats, AISQL);```

Reply:
Done